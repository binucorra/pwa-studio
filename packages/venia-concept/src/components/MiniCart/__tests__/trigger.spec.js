import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import Trigger from '../trigger';

test('renders children inside of a button', () => {
    const props = {
        children: <span>Hi, I'm a child</span>
    };

    const tree = createTestInstance(<Trigger {...props} />).toJSON();

    expect(tree).toMatchSnapshot();
});

test('renders a button with no children when children not supplied', () => {
    const props = {};

    const tree = createTestInstance(<Trigger {...props} />).toJSON();

    expect(tree).toMatchSnapshot();
});
