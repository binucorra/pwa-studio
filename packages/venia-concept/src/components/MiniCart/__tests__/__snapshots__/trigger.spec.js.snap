// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a button with no children when children not supplied 1`] = `<button />`;

exports[`renders children inside of a button 1`] = `
<button>
  <span>
    Hi, I'm a child
  </span>
</button>
`;
